src/components/GuardianIdeal/GuardianIdealCarousel.test.tsx

```diff
@@ -22,10 +22,6 @@ const mockGuardianIdealWidget: Widget = {
 
 describe('GuardianIdealCarousel component', () => {
   test('renders the GuardianIdealCarousel component', () => {
-    mockFlags({
```

Comment:
Discovered when writing my test that we don't need to worry about the flags here, since the flag check happens up a level at MySpace and/or AddWidget